addon/components/plaid-text/template.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if hasBlock}}
+  {{yield}}
```

Comment:
I'm just trying to think of a use case. I'm not sure you can place anything other than a string in a `<text>` tag.